function-images/tests/video-bench/decoder/decode.py

```diff
@@ -0,0 +1,77 @@
+import sys
+import os
+# adding python tracing sources to the system path
+sys.path.insert(0, os.getcwd() + '/../proto/')
+import videoservice_pb2_grpc
+import videoservice_pb2
+
+import io
+import cv2
+import grpc
+import tempfile
+import argparse
+
+from concurrent import futures
+from timeit import default_timer as now
+
+# USE ENV VAR "DecoderFrames" to set the number of frames to be sent
+parser = argparse.ArgumentParser()
+parser.add_argument("-addr", "--addr", dest = "addr", default = "recog.default.192.168.1.240.sslip.io:80", help="recog address")
+parser.add_argument("-sp", "--sp", dest = "sp", default = "80", help="serve port")
+
+args = parser.parse_args()
+
+
+def decode(bytes):
+  temp = tempfile.NamedTemporaryFile(suffix=".mp4")
+  temp.write(bytes)
+  temp.seek(0)
```

Comment:
You should take the video fragment from the request from the video service

Reply:
The video fragment already comes from the request in the video service, see line 82-83:
```
def SendVideo(self, request, context):
    decode(request.value)
```